### ISSUE TITLE
QCEFF RTPS inflate

### ISSUE THREAD

**Issue by hkershaw-brown** [NCAR/DART#748]
In [filter_ensemble_inflate](https://github.com/NCAR/DART/blob/464aa57a1261fa098e5f0f999bad41a464f8d7db/assimilation_code/modules/assimilation/filter_mod.f90#L1549) inflation does transform to/from probit.  

non-RTPS inflation:

```
transform_to_probit
inflate
transform_from_probit
```

https://github.com/NCAR/DART/blob/464aa57a1261fa098e5f0f999bad41a464f8d7db/assimilation_code/modules/assimilation/filter_mod.f90#L1594-L1616

RTPS inflation:

```
inflate
```

https://github.com/NCAR/DART/blob/464aa57a1261fa098e5f0f999bad41a464f8d7db/assimilation_code/modules/assimilation/filter_mod.f90#L1575-L1592

**Comment by hkershaw-brown** [comment]
Need to add RTPS to tests. 
Unclear if it is intentionally not being probit transformed (incompatible?) or an oversight.

**Comment by hkershaw-brown** [comment]
notes from conversation: RTPS is not compatible with QCEFF.

**Comment by jlaucar** [comment]
This issue was discussed in the code review of the QCEFF inflation issue by
Moha yesterday. For now, we do not understand whether RTPS could be
implemented in a transformed space or not. The code in the QCEFF patch will
continue to do RTPS in the same way, whether a probit transform is selected
for the inflation or not. Should this be documented somewhere besides in
the code?

**Comment by hkershaw-brown** [comment]
Documented in the documentation and code corrected, right? The code is silently ignoring QCEFF at the moment. If someone selects RTPS inflation, should qceff be disabled?

I'm assuming the QCEFF patch is this branch: https://github.com/NCAR/DART/tree/qceff_inflation_fix ? So separate issue to RTPS?

**Comment by jlaucar** [comment]
This should be a separate issue. The RTPS can be used any observation space
distribution and with the probit transform for both the state and observed
variables during the assimilation. It is only the inflation specification
that is a no-op.

We are going to need separate issues for this question and for the group
filter in combination with the QCEFF. Lili has used this but we have never
tested it and there are several places where what we're doing in the case
of failed transposes is questionable.

Where would this be best documented?

**Comment by hkershaw-brown** [comment]
Yup, this RTPS issue is its own issue (#748) separate from the other QCEFF issues 
https://github.com/NCAR/DART/issues?q=is%3Aopen+is%3Aissue+label%3AQCEFF

For the group filter combo with QCEFF - the best place to document an issue is to make a new issue in github. Label it QCEFF so someone can look at all the QCEFF issues if needed, and we can close the separate issues as they get fixed.

**Comment by jlaucar** [comment]
Just a quick note that RTPS is spelled at RPTS in issue #748. Only
important because a search of issues for "RTPS" returns nothing. Not sure
if it is trivial, worthwhile, or even possible, to switch that to RTPS.

**Comment by hkershaw-brown** [comment]
thanks @jlaucar they look exactly the same to me, I'll change the title.
